lib/metasploit/framework/data_service/proxy/module_data_proxy.rb

```diff
@@ -0,0 +1,13 @@
+module ModuleDataProxy
+
+  def modules(opts = {})
```

Comment:
It doesn't look like this is ever used. Is this just added in for future functionality?